### ISSUE TITLE
Management: Allow slotting modules

### ISSUE THREAD

**Issue by rajan123456** [rajan123456/uPresent#180]
- At the module level, the start time, end time and duration of lectures should be configurable. 
- This should be held at the dates[] level for allowing individual schedules to be modified later.

**Comment by rajan123456** [comment]
Depends on #154

**Comment by rajan123456** [comment]
Duplicate of #178
